src/Microsoft.FeatureManagement/FeatureFilters/Cron/CronFieldKind.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+namespace Microsoft.FeatureManagement.FeatureFilters.Cron
+{
+    /// <summary>
+    /// Define an enum of all required fields of the Cron expression.
+    /// </summary>
+    internal enum CronFieldKind
+    {
+        /// <summary>
+        /// Field Name: Minute
+        /// Allowed Values: 0-59
+        /// </summary>
+        Minute,
+
+        /// <summary>
+        /// Field Name: Hour
+        /// Allowed Values: 1-12
+        /// </summary>
+        Hour,
+
+        /// <summary>
+        /// Field Name: Day of month
+        /// Allowed Values: 1-31
+        /// </summary>
+        DayOfMonth,
+
+        /// <summary>
+        /// Field Name: Month
+        /// Allowed Values: 1-12 (or use the first three letters of the month name)
+        /// </summary>
+        Month,
+
+        /// <summary>
+        /// Field Name: Day of week
+        /// Allowed Values: 0-7 (0 or 7 is Sunday, or use the first three letters of the day name)
```

Comment:
> 0 or 7 is Sunday

Is this from the standard?

Reply:
This is according to the linux man page https://man7.org/linux/man-pages/man5/crontab.5.html
Some cron implementations may not allow 0 to represent Sunday.